registrations/serializers.py

```diff
@@ -339,6 +421,63 @@ class Meta:
         fields = ["id", "email", "language"]
 
 
+class RegistrationPriceGroupSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(required=False)
+    price_group = serializers.PrimaryKeyRelatedField(
+        queryset=PriceGroup.objects.all(),
+    )
+    price = serializers.DecimalField(
+        required=False, max_digits=19, decimal_places=2, min_value=0
+    )
+    price_without_vat = serializers.DecimalField(
+        required=False, read_only=True, max_digits=19, decimal_places=2
+    )
+    vat = serializers.DecimalField(
+        required=False, read_only=True, max_digits=19, decimal_places=2
+    )
+    vat_percentage = serializers.DecimalField(
+        required=False,
+        max_digits=4,
+        decimal_places=2,
+        min_value=0,
+        max_value=Decimal("99.99"),
+    )
+
+    def validate(self, data):
+        validated_data = super().validate(data)
+
+        errors = {}
+
+        vat_percentage = validated_data.get("vat_percentage")
+        if vat_percentage not in [
+            vat[0] for vat in RegistrationPriceGroup.VAT_PERCENTAGES
+        ]:
+            errors["vat_percentage"] = _("{value} is not a valid choice.") % {
+                "value": vat_percentage
+            }
```

Comment:
This returns `{value} is not a valid choice.` as validation message

Should this be 
```
errors["vat_percentage"] = _("%(value)s is not a valid choice.") % {
                "value": vat_percentage
            }
```

Reply:
Fixed